wordpress-mobile/WordPress-Android
# Track tabChanged and authorFilterSelectionChanged events in post list

Fixes #9651 

Adds tracking of two events on Post List screen
- on tab changed
- on author filter changed

I've also added an unrelated comment to the WPMainActivity as it felt unnecessary to create a separate PR for it. Feel free to update the comment.

To test:
1. My Site
2. Blog Posts
3. Swipe to Drafts (post_list_tab_changed)
4. Click on Published (post_list_tab_changed)
5. Change author selection filter to me (post_list_author_filter_changed)
6. Change author selection filter to Everyone (post_list_author_filter_changed)
7. Rotate your device (nothing should be logged)
7. Check the log cat and make sure the events are being logged correctly 

Merge instructions:
~- mark this PR as ready for review~
- merge the PR
